ic-transport-types/src/lib.rs

```diff
@@ -117,6 +118,36 @@ pub struct ReadStateResponse {
     pub certificate: Vec<u8>,
 }
 
+/// The response from a request to the `call` endpoint.
+// TODO: Inline this enum. No need for it to be public.
```

Comment:
Should this be resolved before release? Arbitrary data structures, even public ones, do not have to go in ic-transport-types, but anything that is part of and deserialized from the replica API can and should.

Reply:
Resolved the stale comment. The enum is used to parse the replica response when it responds `200 - Ok`, however the Accepted variant is manually created if the replica responds with `202 Accepted`.